style.css

```diff
@@ -62,42 +90,22 @@ main {
   width: 15rem;
 }
 
-.task-name-input {
-  width: 25rem;
-}
-
-.task-note-textarea {
-  width: 25rem;
-}
-
-.new-form {
-  font-size: 1.3rem;
-}
-
-/* List */
-.task-list {
-  margin-top: 3rem;
-}
-
-.ul {
-  list-style: none;
-}
-
-li {
-  height: 5rem;
-  font-size: 2rem;
-  margin-bottom: 3rem;
+.add-new {
+  padding: 3rem;
+  height: 10rem;
 }
 
+/* Buttons list */
 .btn-form {
   border: solid 1px;
 }
 
-.btn-cancel {
+.btn-trash {
   background-color: red;
+  margin-left: 20px;
 }
 
-.btn-cancel:active {
+.btn-trash:active {
   background-color: darkred;
```

Comment:
this is not a good practice to use named colors instead of using HEX or RGB :) If you want to use names, establish them earlier with the desired HEX code for example using variables [w3schools](https://www.w3schools.com/css/css3_variables.asp)

Reply:
Those are temporary colors, I'm going to change them soon.